File: mobile/app/src/main/java/com/geneirodan/nextgenmeal/data/OrderDish.kt
package com.geneirodan.nextgenmeal.data

import androidx.annotation.Keep
import kotlinx.serialization.Serializable

@Serializable
@Keep
data class OrderDish(
    val quantity: Int,
    val dishId: Int,
    val dish: Dish,
    val orderId: Int
)

File: server/DataAccess/Entities/OrderDish.cs
namespace DataAccess.Entities;

public class OrderDish
{
    public int Quantity { get; set; }
    public int DishId { get; set; }
    public virtual Dish Dish { get; set; } = null!;
    public int OrderId { get; set; }
    public virtual Order Order { get; set; } = null!;
}
